android/gradlew

```diff
@@ -1,5 +1,21 @@
 #!/usr/bin/env sh
 
+#
+# Copyright 2015 the original author or authors.
```

Comment:
Should this be `Copyright 2015 AWS`?

Reply:
These two files are fully generated by gradle wrappers and untouched after generation.